test/FunctionalTests/Microsoft.OData.Core.Tests/Json/DefaultJsonWriterFromStreamFactoryTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DefaultJsonWriterFromStreamFactoryTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETCOREAPP3_1_OR_GREATER
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.OData.Json;
+using Xunit;
+
+namespace Microsoft.OData.Tests.Json
+{
+    public class DefaultJsonWriterFromStreamFactoryTests

```

Comment:
This is mostly a note to myself: I haven't gone through the tests yet, I'd like to see my other feedback addressed first.